src/test/java/com/linkedin/kmf/services/ConsumeServiceTest.java

```diff
@@ -184,4 +198,36 @@ public void run() {
 
   }
 
+  /**
+   * return consume service metrics.
+   * set up the tags for the metrics
+   * @param consumeService ConsumeService object
+   * @return consume service metrics
+   */
+  private Metrics consumeServiceMetrics(ConsumeService consumeService) {
+    Metrics metrics = consumeService.metrics();
+    tags = new HashMap<>();
+    tags.put(TAGS_NAME, TAG_NAME_VALUE);
```

Comment:
This could be put in the constructor of the test or by having a setup method like this:
```java
@BeforeMethod
public void setup() {
  tags = new HashMap<>();
  tags.put(TAGS_NAME, TAG_NAME_VALUE);
}
```

Reply:
Amended.